### PR TITLE
add contributing file

### PR DESCRIPTION
**Pull request by m-idler**
@xalopp @mmoll review this

### CODE REVIEW

**Comment by xalopp** [comment]
Thank you, for your valuable contribution! +1

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    git clone git@github.com:squizlabs/PHP_CodeSniffer.git
+
+# Step 2
+[Install PHPUnit](https://phpunit.de/getting-started.html)
+
+
+# Step 3
+Fork repository and clone your coding standard repository into "CodeSniffer/Standards/"
+
+    git clone git@github.com:m-idler/symfony-coding-standard.git Symfony
```
Comment (mmoll) [inline CONTRIBUTING.md]: <github_username>?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+# Step 4
+
+Check out a new branch to make your changes on: `git checkout -b <your_new_branch`
```
Comment (mmoll) [inline CONTRIBUTING.md]: >?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+# Step 5
+
+Make sure that your tests are running
```
Comment (mmoll) [inline CONTRIBUTING.md]: green

**Pull request merged**